dll/win32/dbghelp/compat.h

```diff
@@ -367,6 +480,8 @@ PRUNTIME_FUNCTION WINAPI RtlLookupFunctionEntry(ULONG_PTR,DWORD*,UNWIND_HISTORY_
 
 #endif
 
+typedef CONTEXT *PCONTEXT;
```

Comment:
Can't we define that in the first `typedef` where the structure is defined?

Reply:
I could do it if it is required, but honestly, I cannot see a good reason to do so, since it is just a duplication for all platforms.